src/styles/catalog.scss

```
@@ -0,0 +1,25 @@
+.catalog {
+  padding: 50px 0 72px;
+  max-width: $screen-width-l;
+  margin: 0 auto;
+
+  &__container {
+    display: grid;
+    grid-template-columns: repeat(1, $card-size);
+    column-gap: 48px;
+    row-gap: 46px;
+    justify-content: center;
+
+    @media (min-width: 488px) {
+      grid-template-columns: repeat(2, 200px);
```

you can use `$card-size` variable
```suggestion
      grid-template-columns: repeat(2, $card-size);
```